### ISSUE TITLE
MpegTs: Fail to read input

### ISSUE THREAD

**Issue by jason-richards** [MediaArea/MediaInfoLib#452]
Hello!  I just ran into an issue where MediaInfo failed to read a 2820 byte TS.  Here is the line I am implicating:

https://github.com/MediaArea/MediaInfoLib/blob/master/Source/MediaInfo/Multiple/File_MpegTs.cpp#L1421

With this line of code, the minimum size of the TS would be 3008 bytes..  Please let me know if you need more information...

**Comment by JeromeMartinez** [comment]
Right, we use 16 packets for TS synchronization. Is it an issue for you? does 2820-byte TS exist is real?
This is definitely not the best method for synchronization, we could differently but different code so development time, and for us it is enough.

Use case?

**Comment by jason-richards** [comment]
An example use case might be an hls iframe play list, small internal mpegts with metadata, etc.  Others may be running into the issue and just not reporting it.

This synchro problem is interesting because the input is already synchronized (first byte 0x47)... not sure why the file is going into untrusted mode.  (Might be having to do with an EBML check that was added)

[small.zip](https://github.com/MediaArea/MediaInfoLib/files/882415/small.zip)
